src/botservice/azext_bot/custom.py

```diff
@@ -3,15 +3,63 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+import json
+import os
+import shutil
 from knack.util import CLIError
 from knack.log import get_logger
 from azure.mgmt.botservice.models import Bot, BotProperties, Sku
-from azure.cli.command_modules.botservice.custom import create_app, provisionConvergedApp
+from azure.cli.command_modules.botservice.custom import (
+    provisionConvergedApp,
+    get_bot_site_name,
+    publish_app as publish_appv3)
+from azure.cli.command_modules.botservice._webutils import (
+    deploy_arm_template,
+    enable_zip_deploy,
+    get_app_settings,
+    _get_site_credential,
+    _get_scm_url)
 from azure.cli.core._profile import Profile  # pylint: disable=unused-import
 
 logger = get_logger(__name__)
 
 
+def get_app_insights_location(key):
```

Comment:
It might be good to start helper methods you do not want to export with _'s e.g. _get_app_insights_location

Reply:
added in recent commit.